caddyconfig/httpcaddyfile/httptype.go

```diff
@@ -43,6 +43,13 @@ func (st ServerType) Setup(originalServerBlocks []caddyfile.ServerBlock,
 	var warnings []caddyconfig.Warning
 	gc := counter{new(int)}
 
+	// tlsCertTags maps certificate filenames to their tag.
+	// This is used to remember which tag is used for each
+	// certificate files, since we need to avoid loading
+	// the same certificate files more than once, overwriting
+	// previous tags
+	var tlsCertTags = make(map[string]string)
```

Comment:
Instead of initializing this here, we should do so where we use it.

Preserve the comment though, when deleting this line - thanks!

Reply:
This is here to track tags across all the server blocks, so I think this should stay outside all of the server block loops.